### PR TITLE
adding Historical page and refractor analogies types to utils

### PR DESCRIPTION
**Pull request by jamest1903**

### CODE REVIEW

File: webpack/src/utils/analogies.js [hunk 1]
```diff
@@ -15,4 +16,131 @@ export const mbPerUnitData = {
...
+for (let key in kwPerUnitCo2) {
+    analogyNames.co2.push(key);
+}
+for (let key in mbPerUnitData) {
+    analogyNames.data.push(key);
+}
+
+
+export const analogiesCo2 = {
+    marathon: {
```
Comment (jamest1903) [inline webpack/src/utils/analogies.js:35]: Here I change from an array of objects to an object with keys names matching the different types. Before we fetched in array by index 0 -> 5

File: webpack/src/components/Analogies.vue [hunk 2]
```diff
@@ -114,7 +111,7 @@ export default {
   <div class="section" :class="dataType === 'co2' ? 'co2' : 'data'">
     <div class="section-title bold"> {{ t('components.analogies.message') }} </div>
     <el-carousel arrow="always" class="analogies" trigger="click" indicator-position="none" @change="statsIndex">
-      <el-carousel-item v-for="(item, index) in [0,1,2,3,4,5]" :key="index" label="." class="analogy">
+      <el-carousel-item v-for="(item, index) in analogyNames[dataType]" :key="index" label="." class="analogy">
```
Comment (jamest1903) [inline webpack/src/components/Analogies.vue:114]: We now pass in the name like "marathon" instead of an index of 0 to 4 to <analogy>

Comment (slecorne) [inline webpack/src/components/Analogies.vue:114]: please change `key` to not use index

File: webpack/src/components/AppFullpage.vue [hunk 3]
```diff
@@ -41,6 +41,7 @@ export default {
   <div class="wrapper">
     <div data-area="logo"></div>
     <h1 data-area="title">{{ t('appTitle') }}</h1>
+    <div data-area="title" id="logoEquiwatt"></div>
```
Comment (slecorne) [inline webpack/src/components/AppFullpage.vue:44]: can you take a higher resolution logo for the full page? looks blurry

File: webpack/src/composables/history.js [hunk 4]
```diff
@@ -4,7 +4,7 @@ import { retrieveHistoryLayers } from './storage';
 const MAX_HEIGHT = 120;
 
 const layerHeightCo2 = (amount) => {
-  const height = 11 + (amount / 0.9) * MAX_HEIGHT; // min 11px, max 150 px for 900g CO2eq ( 8h laptop consumption no activities ~ 200g)
+  const height = 15 + (amount / 0.9) * MAX_HEIGHT; // min 11px, max 150 px for 900g CO2eq ( 8h laptop consumption no activities ~ 200g)
```
Comment (slecorne) [inline webpack/src/composables/history.js:7]: update comment

**Review by slecorne — CHANGES_REQUESTED** [review]

File: webpack/src/components/Analogies.vue (hunk 2 above)
Comment (jamest1903) [reply webpack/src/components/Analogies.vue:114]: Will change to item

File: webpack/src/components/Analogy.vue [hunk 5]
```diff
@@ -1,128 +1,13 @@
...
-    }
-]
+import { analogiesCo2, analogiesData } from '../utils/analogies'
 
 export default {
   props: {
     layer: {type: Object},
     type: {type: String},
-    index: {type: Number},
+    index: {type: String},
```
[the remark this answers is not in the document]
Comment (jamest1903) [reply webpack/src/components/Analogy.vue:10]: changed to name

**Review by slecorne — APPROVED** [review]

**Pull request merged**